consent/strategy_default.go

```diff
@@ -1039,6 +1039,9 @@ func (s *DefaultStrategy) completeLogout(ctx context.Context, w http.ResponseWri
 
 	_, _ = s.revokeAuthenticationCookie(w, r, store) // Cookie removal is optional
 
+	// Optionally notify the identity provider here:
+	_ = s.r.LogoutIdentity(ctx, lr.Subject)
```

Comment:
Here we would need the user identity and / or session ID to pass along. Do we even have that?

Reply:
For a user request, we can get the session through the Kratos /whoami, but what about the backchannel logoout?

Reply:
Do we have the identity Id?
We could revoke all active sessions via Admin API I guess?